ROADMAP.md

```diff
@@ -99,3 +99,28 @@ A plot showing annualized profit yields as a function of pool size.
 ### Follow-on extensions / analysis
 
 * Use the differing annualized profit yields as an input for an agent-based model to study the possible centralisation of validators in staking pools over time.
+
+## 5. Introduce live data to the model initial states
+
+### Description
+
+The current model uses static datasets for the initial states. An API is used to get the number of active validators and ETH staked balance, as the historical ETH supply and price come from a static CSV that needs to be updated regularly.
+
+Change the initial states from static datasets automatically updated values from a live source.
+
+### What’s required to make it happen
+
+* Update the number of active validators and ETH staked balance from an API to a live source.
+* Update the ETH average gas price from a CSV file to a live source.
+* Update the ETH block rewards from a CSV file to a live source.
+* Update the daily ETH price from a CSV file to a live source.
```

Comment:
I suggest to contextualize these lines a bit more, and possibly just choose a couple that make the most sense:
```
The following State Variables and System Parameters could be refactored to be initialized from a live data source, such as an API or Subgraph:
...
* `eth_price_process` Sytem Parameter: Currently set to the mean ETH price from a static CSV file from Etherscan. This parameter directly influences the profit yields of validators in certain environments, and would benefit from being updated live to reflect the current system state.
```

`ETH average gas price` - suggest referring to the specific Initial State or System Parameter that would need to be updated, how the current value is calculated, and how it could benefit from being live.

Reply:
@BenSchZA thanks for the feedback, added `basefee_process`, it would be nice to check if it makes sense